src/libfetchers/tests/parser.cc

```diff
@@ -0,0 +1,202 @@
+#include "../parser.hh"
+#include "../schema.hh"
+
+#include <gmock/gmock-matchers.h>
+
+using namespace testing;
+
+namespace nix::fetchers {
+    using namespace parsers;
```

Comment:
I think we should deindent these like the rest of the code (and go back and deindent the other indented tests too.